tests/integration_tests/conftest.py

```diff
@@ -67,14 +67,14 @@ def test_table_engine_fixture() -> Iterator[str]:
 # pylint: disable=too-many-branches
 @fixture(scope='session', autouse=True, name='test_client')
 def test_client_fixture(test_config: TestConfig, test_db: str) -> Iterator[Client]:
-    compose_file = f'{Path(__file__).parent}/docker-compose.yml'
+    compose_file = f'{PROJECT_ROOT_DIR}/docker-compose.yml'
     if test_config.docker:
-        run_cmd(['docker-compose', '-f', compose_file, 'down', '-v'])
+        run_cmd(['docker', 'compose', '-f', compose_file, 'down', '-v'])
         sys.stderr.write('Starting docker compose')
-        pull_result = run_cmd(['docker-compose', '-f', compose_file, 'pull'])
+        pull_result = run_cmd(['docker', 'compose', '-f', compose_file, 'pull'])
         if pull_result[0]:
             raise TestException(f'Failed to pull latest docker image(s): {pull_result[2]}')
-        up_result = run_cmd(['docker-compose', '-f', compose_file, 'up', '-d'])
+        up_result = run_cmd(['docker', 'compose', '-f', compose_file, 'up', '-d', 'clickhouse'])
```

Comment:
This is some PyCharm weirdness (does not like `docker-compose`, only works with `docker compose`). Also, could it be a good idea to disable this auto-docker-compose by default? CI does not use it; on a local machine, it's easier to just run `docker compose up -d` beforehand, and that's it. Cause otherwise, without proper healthcheck, it's at least +5 sec of waiting for a single test run (from PyCharm, for example), plus an unnecessary `docker compose down -v` each time.

Reply:
I have no problem with disabling the automatic docker compose start.  (I generally run tests on a local instance  using different ports anyway, because then I don't have to worry about conflicts ClickHouse docker instances started by my projects.)